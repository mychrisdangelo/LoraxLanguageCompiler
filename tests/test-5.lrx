int main()
{
	int a;
	int b;

	b = 0;
	while(a < 20)
	{
		b = b + 1;
		a = a + 2;
	}
	return b;
}