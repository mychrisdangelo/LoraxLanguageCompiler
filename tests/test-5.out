#include "lrxlib.h"
int main();

int main()
{
int a_2 = 0;
int b_2 = 0;
int __tmp_int_7 = 0;
int __tmp_int_6 = 0;
int __tmp_int_5 = 0;
int __tmp_int_4 = 0;
int __tmp_int_3 = 0;
int __tmp_bool_2 = false;
int __tmp_int_1 = 0;
int __tmp_int_0 = 0;

__tmp_int_7 = 0;

b_2 = __tmp_int_7;

goto __LABEL_1;
__LABEL_0:
__tmp_int_5 = 1;

__tmp_int_6 = b_2 + __tmp_int_5;

b_2 = __tmp_int_6;

__tmp_int_3 = 2;

__tmp_int_4 = a_2 + __tmp_int_3;

a_2 = __tmp_int_4;

__LABEL_1:
__tmp_int_1 = 20;

__tmp_bool_2 = a_2 < __tmp_int_1;

if(__tmp_bool_2) goto __LABEL_0;
return b_2;
return __tmp_int_0;

}
